Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -231,7 +231,10 @@ def Runner(env):
         if os.name == 'nt' and qemu_version[0] >= '8':
             import win32console
             std_handle = win32console.GetStdHandle(win32console.STD_INPUT_HANDLE)
-            console_mode = std_handle.GetConsoleMode()
+            try:
+                console_mode = std_handle.GetConsoleMode()
+            except:

```

Comment:
needs to be `except Exception`

https://docs.astral.sh/ruff/rules/bare-except/

Reply:
Thanks. Updated.